packages/client/src/v2-events/features/workqueues/EventOverview/components/useAllowedActionConfigurations.tsx

```diff
@@ -153,16 +153,27 @@ export const actionLabels = {
     id: 'event.birth.action.request-correction.label'
   },
   [ClientSpecificAction.REVIEW_CORRECTION_REQUEST]: {
-    defaultMessage: 'Review',
+    defaultMessage: 'Review correction request',
     description: 'Label for review correction button in dropdown menu',
     id: 'event.action.review-correction.label'
   }
 } as const
 
+const reviewLabel = {
+  id: 'buttons.review',
+  defaultMessage: 'Review',
+  description: 'Label for review CTA button'
+}
+
 interface ActionConfig {
   label: TranslationConfig
   icon: IconProps['name']
+  // onClick is used when clicking an action menu item.
   onClick: (workqueue?: string) => Promise<void> | void
+  // onCta is used when clicking a workqueue CTA button. If its not defined, the onClick will be used.
+  onCtaClick?: (workqueue?: string) => Promise<void> | void
+  // ctaLabel is used on workqueue CTA buttons to override the label
+  ctaLabel?: TranslationConfig
```

Comment:
```suggestion
  /** onClick is used when clicking an action menu item. */
onClick: (workqueue?: string) => Promise<void> | void
  /** onCtaClick is used when clicking a workqueue CTA button. If its not defined, the onClick will be used. */
  onCtaClick?: (workqueue?: string) => Promise<void> | void
  /** ctaLabel is used on workqueue CTA buttons to override the label */
  ctaLabel?: TranslationConfig
```
nitpick: gives hover info